select2-parent/select2/src/main/java/org/wicketstuff/select2/Select2Choice.java

```diff
@@ -77,7 +77,13 @@ public Select2Choice(String id, ChoiceProvider<T> provider)
 		super(id, provider);
 	}
 
-	@Override
+    // needed to be able override the class on Scala
```

Comment:
Maybe it is something I have to set at compiler level? As said this is not a stopper for me because I can always roll my own java class with  onInitialize() overridden